bootstrap/runner.py

```diff
@@ -270,17 +271,26 @@ def create_env_file(self):
     def format_files(self):
         """Format python code generated by cookiecutter."""
         click.echo(info("...formatting the cookiecut python code"))
-        subprocess.run(["black", "-q", f"{self.service_dir.resolve()}"])
+        subprocess.run(  # nosec b603
+            [
+                "/usr/local/bin/python3",
```

Comment:
Are we 100% sure that the `python3` executable is here under any circumstances? Why are absolute commands necessary? It works perfectly well as is.

Reply:
Of course I double checked before submitting this PR.

Here is the command to retrieve the python path executed in the Python Docker image:
```shell
 *  Executing task: docker run --rm -it  python:3.11-slim-bullseye 

Python 3.11.1 (main, Jan 23 2023, 21:18:23) [GCC 10.2.1 20210110] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import sys
>>> sys.executable
'/usr/local/bin/python3'
```
Using absolute command paths mitigates a security risk identified by bandit.
https://bandit.readthedocs.io/en/latest/plugins/b607_start_process_with_partial_path.html

The commands will continue to work as before.

Reply:
I read the linked resource and, even though I understand and agree with it in principle, I fear that many Talos users may find themselves not having all the executables in the exact locations specified here. It is not something related to Docker (which I have not mentioned in my comment), I am worried about users that may launch Talos locally and have those executables in different locations. Can we further investigate how common such a scenario could be, in order to evaluate the costs VS benefits ratio more accurately?

Reply:
I had also checked the paths on Debian and Ubuntu systems but you are right in saying that in other operating systems the paths could be different. I have therefore reverted the introduction of absolute paths. Thanks for the review.